CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- Add `check-direct-connect-virtual-interfaces.rb` to check the status of Direct Connect virtual interfaces
```

Comment:
I will fix this up post acceptance but this should look something like this per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md):
```
## [Unreleased]
### Added
- new script check-direct-connect-virtual-interfaces.rb` to check he status of Direct Connect virtual interfaces (@guikcd)
```